FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/util/V2DemoActivity.kt

```diff
@@ -0,0 +1,84 @@
+package com.microsoft.fluentuidemo.util
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import com.microsoft.fluentui.compose.Scaffold
+import com.microsoft.fluentui.icons.SearchBarIcons
+import com.microsoft.fluentui.icons.searchbaricons.Arrowback
+import com.microsoft.fluentui.theme.FluentTheme
+import com.microsoft.fluentui.theme.token.FluentAliasTokens
+import com.microsoft.fluentui.theme.token.FluentGlobalTokens
+import com.microsoft.fluentui.theme.token.FluentIcon
+import com.microsoft.fluentui.theme.token.FluentStyle
+import com.microsoft.fluentui.theme.token.controlTokens.AppBarSize
+import com.microsoft.fluentui.tokenized.AppBar
+import com.microsoft.fluentui.tokenized.controls.FloatingActionButton
+import com.microsoft.fluentui.tokenized.controls.Label
+import com.microsoft.fluentui.tokenized.menu.Dialog
+import com.microsoft.fluentuidemo.DemoActivity
+import com.microsoft.fluentuidemo.R
+import com.microsoft.fluentuidemo.databinding.V2ActivityComposeBinding
+import java.util.UUID
+
+open class V2DemoActivity : ComponentActivity() {
+    companion object {
+        const val DEMO_TITLE = "demo_title"
+    }
+
+    private var content : @Composable () -> Unit = {}
+
+    fun setActivityContent (content : @Composable () -> Unit) {
+        this@V2DemoActivity.content = content
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        val demoTitle = intent.getSerializableExtra(DEMO_TITLE) as String
```

Comment:
What purpose does demoTitle serve?

Reply:
When we start any V2 Activity, we pass the activity name (which is a Srtring) using intent.putExtra() and then get this string using intent.getSerializableExtra() and store it in demoTitle. Then, we assign this value to the 'title' parameter of AppBar, which basically displays the app bar title. 
In this way, we do not enter the app bar title manually but pick it from the Demo class constructor.